### ISSUE TITLE
Minimized view of a panel should not center contents

### ISSUE THREAD

**Issue by damithc** [MarkBind/markbind#697]
If the panels in the current 'Syntax cheat sheet' page were to be minimized (i.e., using the :x: button), this is what we get:
![image](https://user-images.githubusercontent.com/1673303/52908885-53561600-32b9-11e9-8c8d-16c1b139be57.png)

It's a pity that all conent are centered. If not, we could have used this as the initial view of the cheat sheet More generally, we can use minimized panels as a way to present a more _packed_ overview of the contents

**Comment by yamgent** [comment]
I guess we don't usually expect headers to be multilines.

The code snippets can be wrapped around a `<div>` and use bootstrap's `text-left` class. For example:

```html
<panel header="Quick brown fox <div class='text-left'>`1. One`<br>`2. Two`</div> over lazy dog">
  Content
</panel>
```

![image](https://user-images.githubusercontent.com/3168908/52909114-6e2a8980-32bd-11e9-8c5f-9fcd3bbf7f9d.png)

**Comment by damithc** [comment]
@damithc This kind of headers are usually in a `slot`. But is it possible to not to center by default?

**Comment by yamgent** [comment]
> But is it possible to not to center by default?

We can do that I guess, we can modify the Panel to automatically add the `text-left` class.

**Comment by Chng-Zhi-Xuan** [comment]
@damithc , I did some testing and concluded we are unable to add the `text-left` class to `slot headers`, the author have to manually specify it E.G

```HTML
<panel minimized>
  <p slot="header" class="card-title text-left">
    My multi-line <br>
    slot <br>
    header <br>
  </p>
<panel>
```

We can automatically add it for `header` and `alt` text but you mentioned that multi-line headers are usually inserted using a slot.

If we do not require `header` and `alt` text to be left aligned (on the assumption they are not multi-line), then we do not need to have any enhancements and this issue can be closed.

**Comment by damithc** [comment]
As discussed:
* let's do this:
  >We can automatically add it for header and alt text but you mentioned that multi-line headers are usually inserted using a slot.
* As a separate PR (low priority), we can add `slot` support for `alt`

**Comment by damithc** [comment]
Hang on; the first bullet point doesn't help unless the second one is done, am I right?

**Comment by Chng-Zhi-Xuan** [comment]
It helps if you author multi-line `alt` 

```HTML
<panel alt="my multi-line<br>alt<br>text">
   ...
</panel>
```

As discussed, if both `header` and `alt` text are the same, you can use the current implementation of slots for your needs.

**Comment by damithc** [comment]
In that case, shall we put this issue on hold until there is `slot` support for `alt`?

**Comment by Chng-Zhi-Xuan** [comment]
Yes, that would make sense. 

Specifically for the case whereby the user wants a different text from `header` when minimised, that is richly formatted.

**Comment by jovyntls** [comment]
This issue seems to be fixed as of v4.1.0:

<img width="942" alt="image" src="https://user-images.githubusercontent.com/61113575/219993759-febceb92-7897-4aef-ba33-3c6ec9b7bc3e.png">
